lib/locales/de.js

```diff
@@ -11,7 +11,68 @@
+        'Show closest data on hover': 'Zeige nähste Daten beim Überfahren',                            // components/modebar/buttons.js:157
+        'Snapshot succeeded': 'Snapshot erfolgreich',                                    // components/modebar/buttons.js:66
+        'Sorry, there was a problem downloading your snapshot!': 'Es gab ein Problem beim Herunterladen des Snapshots', // components/modebar/buttons.js:69
+        'Taking snapshot - this may take a few seconds': 'Erstelle einen Snapshot - dies kann einige Sekunden dauern',         // components/modebar/buttons.js:57
+        'Zoom': 'Zoom',                                                  // components/modebar/buttons.js:85
+        'Zoom in': 'Hineinzoomen',                                               // components/modebar/buttons.js:121
+        'Zoom out': 'Herauszoomen',                                              // components/modebar/buttons.js:130
+        'close: ': 'schließen',                                                // traces/ohlc/transform.js:139
+        'trace': 'Datenspur',                                                 // plots/plots.js:439
+        'lat: ': 'Lat.: ',                                                   // traces/scattergeo/calc.js:48
+        'lon: ': 'Lon.: ',                                                  // traces/scattergeo/calc.js:49
+        'q1: ': 'q1',                                                    // traces/box/calc.js:130
+        'q3: ': 'q3',                                                    // traces/box/calc.js:131
+        'source: ': 'Quelle: ',                                                // traces/sankey/plot.js:140
+        'target: ': 'Ziel: ',                                          // traces/sankey/plot.js:141
+        'lower fence: ': 'Untere Schranke',                                           // traces/box/calc.js:134
+        'upper fence: ': 'Obere Schranke: ',                                           // traces/box/calc.js:135
+        'max: ': 'Max.: ',                                                   // traces/box/calc.js:132
+        'mean ± σ: ': 'Mittelwert ± σ: ',                                            // traces/box/calc.js:133
+        'mean: ': 'Mittelwert: ',                                                  // traces/box/calc.js:133
+        'median: ': 'Median: ',                                                // traces/box/calc.js:128
+        'min: ': 'Min.: ',                                                 // traces/box/calc.js:129
+
+        // Missing context
+        'open: ': 'Offen: ',                                                  // traces/ohlc/transform.js:136
+        'high: ': '',                                                  // traces/ohlc/transform.js:137
+        'low: ': '',                                                   // traces/ohlc/transform.js:138
```

Comment:
open, high, low, and close (translated above as schließen) all pertain to the financial chart which in English has the same name (OHLC). These refer to stock prices: the first and last trade (usually within one trading day) and the highest and lowest prices of any trade that day.

Reply:
'Eröffnung: '
'Höchstkurs: '
'Tiefstkurs: '